model/src/main/kotlin/config/FileStorageConfiguration.kt

```diff
@@ -36,4 +37,20 @@ data class FileStorageConfiguration(
      * The configuration of a [LocalFileStorage].
      */
     val localFileStorage: LocalFileStorageConfiguration? = null
-)
+) {
+    /**
+     * Create a [FileStorage] based on this configuration.
+     */
+    fun create(): FileStorage {
```

Comment:
Nit: Just "create" in the context of this class sounds like to create a file storage *configuration*, maybe `createStorage`?

Reply:
Done.